README.md

```diff
@@ -26,57 +28,47 @@ If this repo contains any reference architectures, uncomment the heading below a
 See "Reference architecture" in Authoring Guidelines in the public documentation at
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
 -->
-<!-- ## Reference architectures -->
 
+## Reference architectures
+
+[Maximo Application Suite Deployable Architecture](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/reference-architecture/mas_deployable_architecure.svg)
 
 <!-- This heading should always match the name of the root level module (aka the repo name) -->
 ## terraform-ibm-mas
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
 ```hcl
-
+You can use the modular design of this module to provision Maximo Application Suite Core offering or Maximo Application Suite Core + Manage offering.
+
+module "maximo" {
+
+  source                       = "terraform-ibm-modules/mas/ibm"
+  version                      = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release. For example, 1.6.1
+  cluster_id                   = var.cluster_id
+  deployment_flavor            = var.deployment_flavor # For example, core
+  mas_entitlement_key          = var.mas_entitlement_key
+  mas_instance_id              = var.mas_instance_id # For example, inst1
+  mas_license                  = var.mas_license
+  sls_license_id               = var.sls_license_id
+  uds_contact_email            = var.uds_contact_email
+  uds_contact_firstname        = var.uds_contact_firstname
+  uds_contact_lastname         = var.uds_contact_lastname
+  cluster_config_endpoint_type = var.cluster_config_endpoint_type
+  mas_workspace_id             = var.mas_workspace_id # For example, wrkid1
+  mas_workspace_name           = var.mas_workspace_name # For example, wrkns1
+  pipeline_storage_class       = var.pipeline_storage_class # For example, ibmc-vpc-block-retain-10iops-tier
+  storage_class_rwo            = var.storage_class_rwo # For example, ibmc-vpc-block-retain-10iops-tier
+  storage_class_rwx            = var.storage_class_rwx # For example, ibmc-vpc-file-dp2 (file storage required for db2)
+
+}
 ```
-
 ### Required IAM access policies
```

Comment:
you still need to add this section. I suggest adding this for now but you will need to do a test to confirm it:

> You need the following permissions to run this module.
> - IAM services
>     - **Kubernetes Service** service
>         - `Editor` platform access
>         - `Manager` service access